src/Product/Search.php

```diff
@@ -233,6 +233,18 @@ private function addSearchFilters($selectedFilters, $parent, $idShop)
                         );
                     }
                     break;
+
+                case 'review':
+                    if (isset($selectedFilters['review'])
+                        && (
+                            $selectedFilters['review'][0] !== ''
+                        )
+                    ) {
+                        $this->addReviewFilter(
```

Comment:
You should be able to put more than one review

Reply:
explained in comments bellow.